utils_cv/detection/references/coco_eval.py

```diff
@@ -114,7 +114,9 @@ def prepare_for_coco_segmentation(self, predictions):
 
             rles = [
                 mask_util.encode(
-                    np.array(mask[0, :, :, np.newaxis], order="F")
+                    # Change according to the issue related to mask:
+                    #     https://github.com/pytorch/vision/issues/1355#issuecomment-544951911
+                    np.array(mask[0, :, :, np.newaxis], dtype=np.uint8, order="F")
```

Comment:
Looks good, i'll make a separate issue to do an update on the references.

Reply:
Today I noticed in [the newest version of references](https://github.com/pytorch/vision/blob/v0.5.0/references/detection/coco_eval.py) they fixed the bug.

Reply:
It was fixed in the latest version: https://github.com/pytorch/vision/blob/v0.5.0/references/detection/coco_eval.py#L112